### PR TITLE
refactor!: convert this into the "parent" package for autora, without code

### PR DESCRIPTION
**Pull request by hollandjg**
# Description

Remove code and make this project into a place for handling documentation and dependencies for the AutoRA project

# Type of change

- **refactor**: A code change that neither fixes a bug nor adds a feature
- **docs**: Documentation only changes
- **build**: Changes that affect the build system or external dependencies (i.e., poetry, conda)
- **BREAKING_CHANGE**: remove code and change installation process

ToDo:
- [x] Update contributor guidelines (remove from main ReadMe and then review)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -422,3 +422,68 @@ The release process is automated using GitHub Actions.
...
+
+# How to add new packages
+
+This demonstrates how to add a package published under autora-theorist-example in pyPI in the GitHub repository 
+example-contributor/contributor-theorist
+
+## Add the package as optional dependency
+In the `pyorject.toml` file add an optional dependency for the package in the [project.optional-dependencies] section:
+```toml
+example-theorist = ["autora-theorits-example"]
```
Comment (hollandjg) [inline README.md:434]:
```suggestion
example-theorist = ["autora-theorist-example"]
```

**Review by musslick — APPROVED** [review]
Looks great!

**Review by musslick — APPROVED** [review]
Looks great!

**Review by younesStrittmatter — APPROVED** [review]
looks good

**Review by benwandrew — APPROVED** [review]

**Pull request merged**